cfpb/cfgov-refresh
# Convert link icons to SVG icons

Update all found existence of link icons (other than heading links) to use SVG icons. Includes parsed-links and it's related tests.

Co-authored-by: Scott Cranfill <scott.cranfill@cfpb.gov>

## Changes

- Updated any files currently using link icons to use their SVG equivalent.

## Testing

1. `npm install`
1. `gulp clean && gulp build`
1. Check out external and download parsed links at `http://localhost:8000/practitioner-resources/library-resources/`
1. Check out email link at the bottom of `http://localhost:8000/about-us/the-bureau/bureau-structure/`
1. Check out the external links at the bottom of `http://localhost:8000/about-us/blog/3-common-credit-issues-and-what-you-can-do-fix-them/`
1. Check out the download link at `http://localhost:8000/about-us/careers/management-and-program-analyst-workforce-planning-93mpa/`
1. Check out the download links at `http://localhost:8000/know-before-you-owe/`
1. Check out the download links at `http://localhost:8000/know-before-you-owe/compare/`
1. Check out the download links at `http://localhost:8000/know-before-you-owe/timeline/`
1. Check out the jump links at `http://localhost:8000/your-story/`
1. Check out the external link in the sidebar at `http://localhost:8000/consumer-tools/financial-well-being/about/`

## Screenshots


## Notes

- This is just one part of the SVG Icon update. It's a lot easier to call these out individually than in the main PR.

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
